aten/src/ATen/native/cuda/group_norm_kernel.cu

```diff
@@ -74,7 +78,32 @@ __global__ void ComputeFusedParamsCUDAKernel(
 }
 
 template <typename T>
-__global__ void GroupNormForwardSimpleCUDAKernel(
+__global__ void GroupNorm1dForwardCUDAKernel(
```

Comment:
I wonder if these pw ops wouldn't be better with TensorIterator - it has unrolled loads,